### ISSUE TITLE
Cross icon is confusing in an input element

### ISSUE THREAD

**Issue by alecpl** [twbs/bootstrap#28440]
The icon used on input.is-invalid. There are a few tickets about this issue and some more thumbs up on them, but closed. Please, reconsider.

I also just found the icon confusing. It's usually used as a reset button, and I'm pretty sure IE or Edge uses such icon to reset the input value. Maybe use an exclamation mark.

**Comment by alecpl** [comment]
I really think it should be fixed in v4. It's been added in 4.2 so it's not that v4 has been like that since the beginning. I would call it a regression.

**Comment by Levdbas** [comment]
I must say that a exclamation mark instead of the cross does work pretty well. Icon used in example: 
![afbeelding](https://user-images.githubusercontent.com/17764157/58492269-d6b4bc80-8170-11e9-8b61-584ccbf1218d.png)


![example](https://user-images.githubusercontent.com/17764157/58492095-8ccbd680-8170-11e9-8371-3074402ad7d7.PNG)
